frontend/src/routes/_layout/index.tsx

```diff
@@ -155,7 +155,7 @@ function ExplorePage() {
           },
         ]}
       />
-      <div className="flex flex-col gap-6 md:gap-0 px-6">
+      <div className="flex flex-col gap-6 md:gap-0 px-6 md:max-w-[1440px] mx-auto">
```

Comment:
I think it should be max-w to tablet size -- on my laptop, the 


Also... isn't md and max-w-[1440px] misleading? "Everything bigger than medium (992px) should have a max width of 1440px"

```
<div class="w-full flex justify-center px-5 md:px-0">
  <main class="w-full md:max-w-screen-xl">
  ```
  
  div 1: Full, centered, padded on mobile, no padding on tablet and higher
  div 2: Full width of parent, only as wide as a tablet